lib/re.js

```diff
@@ -101,10 +101,8 @@ module.exports = function (opts) {
       '|' +
       '(?:' + re.src_pseudo_letter + ')' +
       '|' +
-      // don't allow `--` in domain names, because:
-      // - that can conflict with markdown &mdash; / &ndash;
-      // - nobody use those anyway
-      '(?:' + re.src_pseudo_letter + '(?:-(?!-)|' + re.src_pseudo_letter + '){0,61}' + re.src_pseudo_letter + ')' +
+       '(?:' + re.src_pseudo_letter + '(?:-' +
+        '|' + re.src_pseudo_letter + '){0,61}' + re.src_pseudo_letter + ')' +
```

Comment:
This 2 lines have broken formatting/intent - not like other code. Please, follow conding style for other parts of file. All you had to do - just remove `(?!-)` and don't touch anything else.